### DISCUSSION TITLE
Way to fetch artist art, as well as album art?

### DISCUSSION THREAD

**Discussion by riley-martine** [beetbox/beets#4930] [Q&A] [answered]
I recently started using `beets` with the `fetchart` plugin to download missing album art, and it's been very helpful for that. I'd like to also be able to download art for the artists themselves, though. Is there a pre-existing plugin or other way to search for and download artist art?

I searched here and on the old Discourse forums, and didn't see anything, but could've missed it. I also skimmed the `fetchart` code and didn't see anything there.

**Comment by wisp3rwind** [comment, marked as answer]
That's a long standing feature request, but nobody has actually implemented it yet: #889

**Reply by riley-martine** [reply]
Thank you, dumping the script I wrote the other day to do this over there.
